package.json

```diff
@@ -41,8 +41,8 @@
     "lodash.clonedeep": "^4.5.0",
     "lodash.debounce": "^4.0.8",
     "prop-types": "^15.7.2",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
```

Comment:
Change back to 16.14.0
This was changed for a reason

Reply:
And the reason is? Looking at React's changelog, I saw nothing that broke the code significantly

Reply:
Under the hood it changed quite a bit, and Mui for 1 isn't 17 ready. Check the latest stuff on Mui for that one. Ideally we want Mui 5 out as that has more of these changes in

Reply:
I see, I'll revert the React update for this PR and do it on another branch